2024-11-25 18:26:38,602 DEBUG    root            Loaded Command Group: ['gcloud', 'compute']
2024-11-25 18:26:38,614 DEBUG    root            Loaded Command Group: ['gcloud', 'compute', 'instances']
2024-11-25 18:26:38,624 DEBUG    root            Loaded Command Group: ['gcloud', 'compute', 'instances', 'set_service_account']
2024-11-25 18:26:38,626 DEBUG    root            Running [gcloud.compute.instances.set-service-account] with arguments: [--scopes: "['https://www.googleapis.com/auth/cloud-platform']", --service-account: "644941739395-compute@developer.gserviceaccount.com", INSTANCE_NAME: "bash-scripting"]
2024-11-25 18:26:38,681 DEBUG    google.auth.transport.requests Making request: GET http://metadata.google.internal/computeMetadata/v1/instance/service-accounts/644941739395-compute@developer.gserviceaccount.com/?recursive=true
2024-11-25 18:26:38,685 DEBUG    urllib3.connectionpool Starting new HTTP connection (1): metadata.google.internal:80
2024-11-25 18:26:38,686 DEBUG    urllib3.connectionpool http://metadata.google.internal:80 "GET /computeMetadata/v1/instance/service-accounts/644941739395-compute@developer.gserviceaccount.com/?recursive=true HTTP/1.1" 200 197
2024-11-25 18:26:38,687 DEBUG    google.auth.transport.requests Making request: GET http://metadata.google.internal/computeMetadata/v1/instance/service-accounts/644941739395-compute@developer.gserviceaccount.com/token
2024-11-25 18:26:38,690 DEBUG    urllib3.connectionpool http://metadata.google.internal:80 "GET /computeMetadata/v1/instance/service-accounts/644941739395-compute@developer.gserviceaccount.com/token HTTP/1.1" 200 866
2024-11-25 18:26:52,396 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): compute.googleapis.com:443
2024-11-25 18:26:52,440 DEBUG    urllib3.connectionpool https://compute.googleapis.com:443 "POST /compute/v1/projects/inspired-photon-441609-k7/zones/us-central1-a/instances/bash-scripting/setServiceAccount?alt=json HTTP/1.1" 403 None
2024-11-25 18:26:52,444 DEBUG    root            Chosen display Format:list[title="Could not fetch resource:",always-display-title]
2024-11-25 18:26:52,445 DEBUG    root            (gcloud.compute.instances.set-service-account) Could not fetch resource:
 - Request had insufficient authentication scopes.
Traceback (most recent call last):
  File "/snap/google-cloud-cli/275/lib/googlecloudsdk/calliope/cli.py", line 998, in Execute
    resources = calliope_command.Run(cli=self, args=args)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/snap/google-cloud-cli/275/lib/googlecloudsdk/calliope/backend.py", line 842, in Run
    resources = command_instance.Run(args)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/snap/google-cloud-cli/275/lib/surface/compute/instances/set_service_account.py", line 158, in Run
    return client.MakeRequests([(
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/snap/google-cloud-cli/275/lib/googlecloudsdk/api_lib/compute/client_adapter.py", line 145, in MakeRequests
    utils.RaiseToolException(
  File "/snap/google-cloud-cli/275/lib/googlecloudsdk/api_lib/compute/utils.py", line 139, in RaiseToolException
    RaiseException(problems, calliope_exceptions.ToolException, error_message)
  File "/snap/google-cloud-cli/275/lib/googlecloudsdk/api_lib/compute/utils.py", line 148, in RaiseException
    raise exception(
googlecloudsdk.calliope.exceptions.ToolException: Could not fetch resource:
 - Request had insufficient authentication scopes.

2024-11-25 18:26:52,447 ERROR    root            (gcloud.compute.instances.set-service-account) Could not fetch resource:
 - Request had insufficient authentication scopes.

